### ISSUE TITLE
Doesn't work on (my) Mojave even with this accessibility thing

### ISSUE THREAD

**Issue by kud** [milgra/macmediakeyforwarder#64]
Hello,

All is in the title. :/

**Comment by kud** [comment]
<img width="324" alt="image" src="https://user-images.githubusercontent.com/655838/48938405-64d58f00-ef11-11e8-9887-42e882ae8a4b.png">

**Comment by agentspooky** [comment]
I've had the same experience. When MacMediaKeyForwarder is enabled, my media keys NEVER control iTunes; when it is disabled, my media keys follow the expected (stupid) macOS behavior. Yes, I've enabled Accessibility control.

**Comment by irvinlim** [comment]
Not a macOS developer, but I tried cloning the repo and running the application on Mojave from AppCode (or alternatively, Xcode), and got the prompt for Apple Events permissions from AppCode:

![screenshot 2018-11-25 at 3 54 08 pm](https://user-images.githubusercontent.com/9884746/48976891-60080b00-f0ca-11e8-99c5-e5e690a0642d.png)

After clicking "Allow", media key presses are forwarded correctly for me on Mojave. The relevant entry subsequently appears under _System Preferences > Security & Privacy > Automation_:

![screenshot 2018-11-25 at 3 53 22 pm](https://user-images.githubusercontent.com/9884746/48976890-4961b400-f0ca-11e8-9e55-14b847599087.png)

Perhaps MacMediaForwarder needs to provide a way to trigger this popup using the [Apple Events API](https://developer.apple.com/documentation/applicationservices/apple_event_manager)?
